spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigurationChangeDetector.java

```diff
@@ -77,36 +78,34 @@ public void reloadProperties() {
 
 	/**
 	 * Determines if two property sources are different.
-	 * @param mp1 map property sources 1
-	 * @param mp2 map property sources 2
+	 * @param left left map property sources
+	 * @param right right map property sources
 	 * @return {@code true} if source has changed
 	 */
-	protected boolean changed(MapPropertySource mp1, MapPropertySource mp2) {
-		if (mp1 == mp2) {
+	protected boolean changed(MapPropertySource left, MapPropertySource right) {
+		if (left == right) {
 			return false;
 		}
-		if (mp1 == null && mp2 != null || mp1 != null && mp2 == null) {
+		if (left == null || right == null) {
 			return true;
 		}
-
-		Map<String, Object> s1 = mp1.getSource();
-		Map<String, Object> s2 = mp2.getSource();
-
-		return s1 == null ? s2 != null : !s1.equals(s2);
+		Map<String, Object> leftMap = left.getSource();
+		Map<String, Object> rightMap = right.getSource();
+		return !Objects.equals(leftMap, rightMap);
 	}
 
-	protected boolean changed(List<? extends MapPropertySource> l1,
```

Comment:
I _think_ that having names as `left/right` simplifies reading the code. For example, this is what JDK code does in `Collectors::toList`, but not limited to.